lib/kitchen/driver/azurerm.rb

```diff
@@ -75,7 +75,7 @@ class Azurerm < Kitchen::Driver::Base
       end
 
       default_config(:password) do |_config|
-        SecureRandom.base64(12)
+        SecureRandom.base64(24)
```

Comment:
will 24 continue to work on older windows releases?

Reply:
How old are we talking? I believe post Windows 98 password support is greater than 16 characters up to 128 (even 256 with modification) characters. I am trying to confirm with documentation versus past experience but I do believe Windows NT and 98 and before only supported up to 14 characters. We're not targeting those versions of Windows are we?